README.md

```diff
@@ -61,6 +61,8 @@ $ go run . --help
     	path to an artifact to verify
   -branch string
     	expected branch the binary was compiled from (default "main")
+  -output-provenance
```

Comment:
shall we rename to `--display-provenance` or `--print-provenance`. `output-provenance` may be interpreted as a file to output the provenance.
Otherwise, shall we rename `--provenance` to `input-provenance`

I think the former looks better. Wdut?

Reply:
Oh, changed to `--print-provenance`

Reply:
LGTM